### PR TITLE
net-mgmt/lldpd: Add auto-refresh for neighbor tab

### PR DESCRIPTION
**Pull request by mimugmail**
Add auto-refresh, thanks to @fabianfrz :)

@fichtner I think this can go directly to RC2, but you can decide better about timeline etc.

### CODE REVIEW

File: net-mgmt/lldpd/src/opnsense/mvc/app/views/OPNsense/Lldpd/general.volt [hunk 1]
```diff
@@ -54,9 +62,9 @@ $( document ).ready(function () {
     ajaxCall(url="/api/lldpd/service/status", sendData={}, callback=function(data,status) {
         updateServiceStatusUI(data['status']);
     });
-    ajaxCall(url="/api/lldpd/service/neighbor", sendData={}, callback=function(data,status) {
```
Comment (fichtner) [inline net-mgmt/lldpd/src/opnsense/mvc/app/views/OPNsense/Lldpd/general.volt]: Zap the other one below too?

Comment (mimugmail) [reply net-mgmt/lldpd/src/opnsense/mvc/app/views/OPNsense/Lldpd/general.volt]: How would it be called .. might be to stupid for this :)

Comment (fichtner) [reply net-mgmt/lldpd/src/opnsense/mvc/app/views/OPNsense/Lldpd/general.volt]: the update in the reconfigure block that I added and which started the whole discussion ;)

Comment (mimugmail) [reply net-mgmt/lldpd/src/opnsense/mvc/app/views/OPNsense/Lldpd/general.volt]: oh ... ok, removed :)

**Comment by fichtner** [comment]
ok, thank you <3

**Pull request merged**